docs/terms.md

```diff
@@ -0,0 +1,142 @@
+## Suite
```

Comment:
Maybe a comment on top describing the hierarchical structure of things with suite, test spec, test and params?